src/neural/cuda/common_kernels.cu

```diff
@@ -553,16 +553,22 @@ void OutputInputTransform(int N, int C, int se_K, T* output, const T* input,
                           const T* b1, const T* w2, const T* b2,
                           cudaStream_t stream) {
   // Each thread processes entire chess board
-  if (C > kMaxResBlockFusingChannels) {
-      throw Exception(
-          "res block fusing opt not supported for the given data type and no "
-          "of filters\n");
+  if (use_se == false) {
+    dim3 grid_dim(DivUp(C, kOpInpTransformBlockSize), N, 1);
+    OutputTransform_relu_InputTransform_kernel<float, relu, use_bias, use_skip>
+        <<<grid_dim, kOpInpTransformBlockSize, 0, stream>>>(N, C, output, input,
+                                                            (float*)skip, bias);
+  } else if (C > kMaxResBlockFusingChannels) {
+    throw Exception(
+        "res block fusing opt not supported for the given data type and no "
+        "of filters\n");
   } else {
     OutputTransform_SE_relu_InputTransform_kernel<float, use_se, relu, use_bias,
```

Comment:
This kernel will always be called with use_se=true, so some additional simplification is possible.

Reply:
Thanks. I have adjust the change appropriately (after resolving conflicts). This also fixes the perf regression caused by the MISH change (due to running out of registers and local memory spills for use_se==false case). I also got rid of 8x8 skipInp array for the use_se==true kernel to reduce register pressure a bit (which also fixes the random functional failure that we were seeing after the activation change on some GPUs).